htlcswitch/switch.go

```diff
@@ -723,9 +723,10 @@ func (s *Switch) htlcForwarder() {
 
 			// Otherwise, we'll log this diff, then accumulate the
 			// new stats into the running total.
-			log.Infof("Sent %v satoshis received %v satoshi "+
+			log.Infof("Sent %v satoshis received %v satoshis "+
 				"in the last 10 seconds (%v tx/sec)",
```

Comment:
Well, I didn't change that. I think if we want to have a policy on decimal places in logging, there'll be a lot of different places that need a change.